source/tutorial/create-system-dsn.txt

```diff
@@ -62,25 +182,25 @@ Procedure
        content: |
          .. include:: /includes/steps/create-system-dsn.rst
 
-         Example
-         -------
-
-         The following example configuration assumes the following:
-
-         - A :binary:`~bin.mongosqld` instance running on ``localhost`` port
-           ``3307`` with authentication enabled.
-         - A :manual:`mongod </reference/program/mongod/>` instance running with
-           :manual:`authentication </reference/program/mongod#cmdoption-auth>` enabled.
-         - A :manual:`MongoDB user </core/security-users/>` named
-           ``dbUser`` who authenticates against the ``admin`` database
-           and is authorized to use the ``test`` database.
-
-         The values you enter may be different.
-
-         .. figure:: /images/bi-connector/windows-odbc-dsn.png
-            :alt: Screenshot of the Windows ODBC Administrator application
-            :figwidth: 550px
-            :align: center
+         .. example::
+
+            The following example configuration assumes the following:
+   
+            - A :binary:`~bin.mongosqld` instance running on 
+              ``localhost`` port ``3307`` with authentication enabled.
+            - A :manual:`mongod </reference/program/mongod/>` instance 
+              running with Username and Password (``SCRAM-SHA-1``)
+              :manual:`authentication </reference/program/mongod#cmdoption-auth>` enabled. 
```

Comment:
I don't think this is a good link. Maybe link here?

https://docs.mongodb.com/manual/core/security-scram/#authentication-scram

Reply:
This points to authentication, not to SCRAM-SHA-1. It works for that purpose.